configure.ac

```diff
@@ -591,6 +591,33 @@ AC_MSG_RESULT(${enable_builtin_mbedtls})
 AM_CONDITIONAL([OPENTHREAD_ENABLE_BUILTIN_MBEDTLS], [test "${enable_builtin_mbedtls}" = "yes"])
 
 #
+# Debug UART
+#
+AC_MSG_CHECKING([whether to enable the DEBUG UART])
+AC_ARG_ENABLE(debug_uart,
+    [AS_HELP_STRING([--enable-debug-art],[Enable DEBUG UART support @<:@default=no@:>@.])],
+    [
+        case "${enableval}" in
+
+        no|yes)
+            enable_debug_uart=${enableval}
+            ;;
+
+        *)
+            AC_MSG_ERROR([Invalid value ${enable_debug_uart} for --enable-debug-uart])
+            ;;
+        esac
+    ],
+    [enable_debug_uart=no])
+
+if test "$enable_debug_uart" = "yes"; then
```

Comment:
Suggestion: Can we consider adding this as a `OPENTHREAD_CONFIG` option in `openthread-core-default-config.h` instead of it being a `configure.ac` option?